DataFormats/L1TrackTrigger/interface/L1Track.h

```diff
@@ -0,0 +1,11 @@
+#ifndef DataFormats_L1TrackTrigger_L1Track_h
+#define DataFormats_L1TrackTrigger_L1Track_h
+
+#include "DataFormats/L1TrackTrigger/interface/TTTrack.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTypes.h"
+#include <vector>
+
+using L1Track = TTTrack<Ref_Phase2TrackerDigi_>;
```

Comment:
it was pointed out by Slava that this name is generic enough and might be more safer in a namespace, to avoid potential future symbol conficts. This is not strictly a reco file, so I let someone else give their suggestion as to what this could be.

Reply:
sure on this, I honestly have no preference, perhaps @rekovic  could comment on what the L1 prefers here?

For what its worth on this point, I agree. I would have stuck it in whatever namespace TTrack is in but its in none.

Reply:
IIRC there is an `l1t` namespace.
Perhaps we can place it there?
The full name `l1t::L1Track` is a bit redundant though.

BTW, are there any use cases of `TTTrack` instances? How would those be named with a typedef in a similar pattern?

Reply:
I dont believe there are any other such instances after having a brief but not exhaustive look.  I'm fine with l1t::L1Track, we could also have l1t::TTTrack but given most code using a TTTrack is likely in the l1t namespace, its probably a bad idea. Last option would be l1t::Phase2Track which is a bit longer but also acceptable to me.  I really have little preference here so if people have strong feelings I'm happy to change it to whatever. 

 I'll give it a little while to see if Vladimir comments.

Reply:
I should clarify one point: my main concern in the earlier discussion with @jpata was a possible conflict in compiled symbols. With a bit more thought, a typedef is just a synonym and a conflict if any will be at compile time and will be easily manageable.
So, perhaps there is no problem after all.